roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -26,6 +14,21 @@
   - { site: "sthlm", project_id: "{{ ngi_pipeline_sthlm_delivery }}" }
   - { site: "upps", project_id: "{{ ngi_pipeline_upps_delivery }}" }
 
+- name: Create R dependencies subdirectory
+  file: path="{{ ngi_rnaseq_dest }}/r_packages" state=directory mode=g+rwXs
+
+- name: Copy R dependencies script
+  copy:
+    src: "files/install_R_dependencies.r"
+    dest: "{{ ngi_rnaseq_dest }}/r_packages"
+
+- name: Run R script to install dependencies
+  command: "Rscript install_R_dependencies.r"
+  environment:
+    R_LIBS_USER: "{{ ngi_rnaseq_dest }}/r_packages"
+  args:
+    chdir: "{{ nextflow_dest }}/r_packages"
```

Comment:
My bad seems like I missed this to change yesterday, but I believe it should be `"{{ ngi_rnaseq_dest }}/r_packages"` ? Also since you are at it could change [this](https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/master/roles/ngi-rnaseq/templates/ngi-rnaseq_site.config#L3) for me. Tack 😇